src/client.ts

```diff
@@ -111,18 +114,64 @@ export class Client {
     }
   };
 
+  private _onKeyDown = (event: KeyboardEvent) => {
+    if (!this._registeredKeys) {
+      return;
+    }
+
+    const keyData = Key.fromKeyEvent(event);
+    const shouldSend = this._registeredKeys.some(
+      (key: string) => key === keyData.serialize()
```

Comment:
Serialization seems like an unnecessary extra step for comparison of the events. It's just more code to ship to the browser and extra logic where bugs could creep in.  Seems like just comparing the individual fields from the key registration data should be sufficient.

Reply:
Yeah the thought was that string comparison should be less performance intensive than multiple property comparison since we could serialize the registered keys at startup. But yeah it is more code and maybe reducing the load on v8 would result in faster speeds. I went ahead and changed this